app/models/reviewable_score.rb

```diff
@@ -11,7 +11,7 @@ class ReviewableScore < ActiveRecord::Base
   # To keep things simple the types correspond to `PostActionType` for backwards
   # compatibility, but we can add extra reasons for scores.
   def self.types
-    @types ||= PostActionType.flag_types.merge(needs_approval: 9)
+    @types ||= PostActionType.flag_types.merge(needs_approval: 9).merge(PostActionType.score_types)
```

Comment:
Will we be able to get rid of this weird `merge(needs_approval: 9)` at some point?

Reply:
oh yes! since we have score type I can add it to fixtures and have that in the database!